be/src/vec/functions/function.h

```diff
@@ -54,6 +54,11 @@ namespace doris::vectorized {
     return is_nullable || !is_datev2 ? make_nullable(std::make_shared<TYPE>())           \
                                      : std::make_shared<TYPE>();
 
+#define SET_NULLMAP_IF_FALSE(EXPR) \
```

Comment:
This marco is overly specialized, it can even just be used in this pr.

Reply:
yes for now. but it's useful and could be used to refactor many code in future. so I leave them in `function.h`